doc/api/stream.md

```diff
@@ -1754,8 +1754,9 @@ methods only.
 
 The `writable._writev()` method may be implemented in addition to
 `writable._write()` in stream implementations that are capable of processing
-multiple chunks of data at once. If implemented, the method will be called with
-all chunks of data currently buffered in the write queue.
+multiple chunks of data at once. If implemented, the first chunk will be written with 
+`_write()`. When the initial write is complete, buffered chunks in the write queue, if available, 
```

Comment:
I think it’s not just the first chunk, but rather, `_write()` is always used when there is only one chunk in the queue?

Reply:
When the queue is empty.

Reply:
Isn't current docs correct in that case @addaleax @lpinca  ? Though still bit ambiguous. Does below seems more correct -

>  if implemented, **only the `writable._writev()`** method will be called with all chunks of data currently buffered in the write queue. In case there is only a single data chunk to be processed **`writable._writev()`** is called instead.

Reply:
@erabhimanyu I think your suggestion is still confusing. In particular this sentence

> In case there is only a single data chunk to be processed `writable._write()` is called instead

If there is buffered data even a single chunk, `writable._writev()` will be used.